components/tag-list/tag-list-item-mixin.js

```diff
@@ -187,55 +190,72 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
+			<ul>
+				<li><span class="d2l-tag-list-item-tooltip-title-key">${this.localize('components.tag-list-item.tooltip-arrow-keys')}</span> - ${this.localize('components.tag-list-item.tooltip-arrow-keys-desc')}</li>
+				<li><span class="d2l-tag-list-item-tooltip-title-key">${this.localize('components.tag-list-item.tooltip-delete-key')}</span> - ${this.localize('components.tag-list-item.tooltip-delete-key-desc')}</li>
+			</ul>
+		`;
 	}
 
-	_handleTooltipShow() {
-		this._tooltipShown = true;
-		/** @ignore */
-		this.dispatchEvent(new CustomEvent(
-			'd2l-tag-list-item-tooltip-show',
-			{ bubbles: true, composed: true }
-		));
-	}
+	_renderTag(tagContent, options) {
+		if (!options) options = {};
 
-	_renderTag(tagContent, hasTruncationTooltip, description) {
 		const buttonText = typeof tagContent === 'object'
 			? this.localize('components.tag-list.clear', { value: '' })
 			: this.localize('components.tag-list.clear', { value: tagContent });
-		const hasDescription = !!description;
-		const tooltipHeader = hasDescription ? html`<div class="d2l-heading-4">${tagContent}</div>` : tagContent;
-		const tooltipTagOverflow = hasTruncationTooltip ? html`
+
+		const hasDescription = !!options.description;
+
+		let tooltip = nothing;
+		if (this._displayKeyboardTooltip) {
+			tooltip = html`
+				<d2l-tooltip
+					align="start"
+					@d2l-tooltip-hide="${this._handleKeyboardTooltipHide}"
+					@d2l-tooltip-show="${this._handleKeyboardTooltipShow}"
+					for="${this._id}">
+						${this._renderKeyboardTooltipContent()}
+				</d2l-tooltip>`;
+		} else if (options.hasTruncationTooltip || hasDescription) {
+			const tooltipHeader = hasDescription ? html`<div class="d2l-heading-4">${tagContent}</div>` : tagContent;
+			tooltip = html`
 				<d2l-tooltip for="${this._id}" ?show-truncated-only="${!hasDescription}">
 					${tooltipHeader}
-					${hasDescription ? description : nothing}
-				</d2l-tooltip>
-			` : null;
-		const tooltipKeyboardInstructions = this._displayKeyboardTooltip ? html`
-			<d2l-tooltip
-				align="start"
-				@d2l-tooltip-hide="${this._handleTooltipHide}"
-				@d2l-tooltip-show="${this._handleTooltipShow}"
-				for="${this._id}">
-					${this._renderTooltipContent()}
-			</d2l-tooltip>` : null;
+					${hasDescription ? options.description : nothing}
+				</d2l-tooltip>`;
+		}
+
 		const containerClasses = {
 			'd2l-label-text': true,
 			'tag-list-item-container': true,
 			'tag-list-item-container-clearable': this.clearable
 		};
+		const focusableClasses = {
+			'tag-list-item-content': true
+		};
+		if (options.focusableClass) focusableClasses[options.focusableClass] = true;
+
 		return html`
-			${tooltipKeyboardInstructions || tooltipTagOverflow}
+			${tooltip}
 			<div class="${classMap(containerClasses)}">
-				<div class="tag-list-item-content" id="${this._id}" tabindex="-1">${tagContent}</div>
+				<div aria-label="${ifDefined(options.label)}"
```

Comment:
This element will get the focus. In order for the tooltips to be announced correctly, it must have an interactive role. To make the semantics clearer, `aria-roledescription` is used.

The addition of `aria-label` option is required by the user's tag-list-item, since it provides some special instruction for screen reader users.

Reply:
Cool, hadn't seen `aria-roledescription` before!